google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -54,10 +54,16 @@ def build(self):
         for scanner in self.scanner_configs.get('scanners'):
             if scanner.get('enabled'):
                 module_path = 'google.cloud.forseti.scanner.scanners.{}'
-                module_name = module_path.format(
-                    scanner_requirements_map.REQUIREMENTS_MAP
-                    .get(scanner.get('name'))
-                    .get('module_name'))
+
+                scanner_name = scanner.get('name')
+                scanner_def = scanner_requirements_map.\
```

Comment:
nit: in python, there is a simpler way to check if a key is in a map (or not in a map) with `in` construct.
This allows you to make no change to the previous code.
```
if not scanner.get('name') in scanner_requirements_map.REQUIREMENTS_MAP:
    log and continue

module_name = module_path.format(	
    scanner_requirements_map.REQUIREMENTS_MAP
    .get(scanner.get('name'))
```

Reply:
I created new variables so that scanner.get('name') didn't need to be run twice in the if statement and the later module_name definition. but it also makes sense to make minimal change to previous code. modified as suggested